_data/apiConfig.json

```diff
@@ -63,14 +111,14 @@
   {
     "doc": "RSA private key to encrypt sensitive configuration parameters with.",
     "docExample": "rsaPrivateKey: \"-----BEGIN RSA PRIVATE KEY-----\\nkey\\n-----END RSA PRIVATE KEY-----\"",
-    "default": null,
+    "default": "",
     "env": "RSA_PRIVATE_KEY",
```

Comment:
For me, this is a bit confusing. User will be creating a `rsaprivatekey` in heroku deployment, and will provide the private key contents there. What about the `RSA_PRIVATE_KEY`? Where is it expected to be used?

Reply:
I'm not quite sure what you mean, but I'll try to clarify. This default empty string is just to avoid an exception. The user is still expected to supply this RSA_PRIVATE_KEY, either as an environment variable (i.e. heroku vars) or in the config json directly. 

It's needed when you have encrypted secrets in the site config such as your recaptcha key.